news/2022-03-12-new-featured-artist-raimukun.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "New Featured Artist: Raimukun"
+date: 2022-03-12 09:00:00 +0000
+---
+
+**Raimukun** crashes into our Featured Artist library!
+
+![](https://assets.ppy.sh/artists/247/header.jpg)
+
+**Raimukun** is part of a new generation of rhythm game song producers. Taking influence from legends like [*xi*](https://osu.ppy.sh/beatmaps/artists/727) and *Sakuzyo* when learning to compose his tunes, **Raimukun** aims to replicate their intensity while also challenging himself through experimentation.
+
+**7** distinctly powerful tracks are now available from [**Raimukun**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/247), including his iconic *BMS OF FIGHTERS* submissions!
+
+The newest of those submissions is [Firmament star](https://osu.ppy.sh/beatmapsets/1633703) (rendered for osu!catch by [Jemzuu](https://osu.ppy.sh/users/7890134)). If you're one of the 3 people who've ever wondered how an osu!catch map coverts to circle clicking, this video will satisfy all of your curiosities. Once you're done, check out everything else below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/247/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Firmament star
+
+Explore [the map from the video above](https://osu.ppy.sh/beatmapsets/1633703) hosted by [Jemzuu](https://osu.ppy.sh/users/7890134)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Icyxis
+
+If you'd rather smash a piano, try [this multi-speed osu!mania map](https://osu.ppy.sh/beatmapsets/1256193) hosted by [11Bit](https://osu.ppy.sh/users/14804526)!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Isysmonahy
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+[**Raimukun**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/247) is the only place you need to be right now. Take a crack at mapping with any of our super convenient pre-timed beatmap templates, then consider why you hadn't started learning to map earlier.
```

Comment:
being in a place makes a bit more sense than being a place

```suggestion
[**Raimukun**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/247) is the only place you need to be in right now. Take a crack at mapping with any of our super convenient pre-timed beatmap templates, then consider why you hadn't started learning to map earlier.
```

Reply:
this is a pretty common exaggeration colloquialism used in the us/au/uk (normally referring to like touristy destinations or other concert/event type things) despite your edit being more semantically correct

Reply:
i have been educated